src/vault_backup/backup.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+import json
+import logging
+
+from lib.export_backup import (
+  backup_to_s3,
+  write_file
+)
+from lib.util import url_host_port
+from lib.vault import Vault
```

Comment:
Our Vault client is packaged in [fortress](https://github.com/artsy/fortress/) now... is there a reason why we need to duplicate that code in this repo? We can just load the fortress dependency in this project.

Reply:
feels like this would be quite a fair amount of work. i am on the fence about this. should we address it now or punt?